### DISCUSSION TITLE
Move menu item to bottom in drawer

### DISCUSSION THREAD

**Discussion by jaimeiniesta** [saadeghi/daisyui#2762] [Q&A] [answered]
Hi, I'm integrating the Daisy UI Dashboard template and I'd like to have one of the menu items stick at the bottom.

That is, instead of having the user dropdown at the top right, I want to move it to the drawer and have it stick at the bottom:

![Captura de pantalla 2024-01-15 a las 15 03 24](https://github.com/saadeghi/daisyui/assets/2629/510c53d3-295e-4b81-81be-e87ebf758d43)

Something like the user dropdown in X:

<img width="314" alt="Captura de pantalla 2024-01-15 a las 15 01 11" src="https://github.com/saadeghi/daisyui/assets/2629/52962021-e4dd-4e91-b946-07a5248a38eb">

What's the best approach for this?

**Comment by saadeghi** [comment, marked as answer]
It's not just moving, it has a little bit different design.

You can put something like this:
```html
<div class="dropdown dropdown-top m-2">
  <div tabindex="0" role="button" class="btn btn-ghost btn-block">
    <div class="avatar w-8">
      <div class="rounded-full"><img alt="" src="https://daisyui.com/images/stock/photo-1534528741775-53994a69daeb.jpg" /></div>
    </div>
    <div class="grow text-start">Name Here</div>
    <svg xmlns="http://www.w3.org/2000/svg" width="16" height="16" viewBox="0 0 24 24" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round">
      <circle cx="12" cy="12" r="1"></circle>
      <circle cx="12" cy="5" r="1"></circle>
      <circle cx="12" cy="19" r="1"></circle>
    </svg>
  </div>
  <ul tabindex="0" class="dropdown-content menu z-[1] my-2 w-full rounded-box bg-base-100 p-2 text-base-content shadow">
    <li><a>Item 1</a></li>
    <li><a>Item 2</a></li>
  </ul>
</div>

```
after `<ul class="menu">…….</ul>` and add `grow` class to that `<ul>` so it fills the empty space.

Let me know if you have a question.

**Reply by jaimeiniesta** [reply]
It works great, thanks!
